extension/content/components/recipes/form/RecipeFormHeader.js

```diff
@@ -30,10 +29,11 @@ export default function RecipeFormHeader() {
     }
   };
 
-  const saveRecipe = () => {
-    const { action } = data;
+  const saveRecipe = (comment) => {
+    const { action, comment: _omitComment, ...rest } = data;
```

Comment:
`rest` is a bit of a ambiguous variable name. i would go with something like `cleanedData`.